### ISSUE TITLE
IDISP001 False Positive with MemoryStream and File Return

### ISSUE THREAD

**Issue by ondrejholik** [DotNetAnalyzers/IDisposableAnalyzers#535]
```csharp
var imageStream = new MemoryStream();
return File(imageStream, "image/png");
```
Should not trigger IDISP001

https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.file?view=aspnetcore-7.0&viewFallbackFrom=net-7.0
